ClosedXML/Utils/CryptographicAlgorithms.cs

```diff
@@ -37,7 +37,7 @@ public static String GetPasswordHash(Algorithm algorithm, String password, Strin
             if (salt == null)
                 throw new ArgumentNullException(nameof(salt));
 
-            if ("" == password) return "";
+            if (string.IsNullOrEmpty(password)) return "";
```

Comment:
same as before (null is now treated differently) but different handling  (no Exception)

Reply:
changed to:
if (password?.Length == 0) return "";